pkg/proxy/proxy.go

```diff
@@ -90,76 +111,48 @@ func (p *Proxy) unlockVIN(vin string) {
 
 // New creates an http proxy.
 //
-// Vehicles must have the public part of skey enrolled on their keychains. (This is a
-// command-authentication key, not a TLS key.)
-func New(ctx context.Context, skey protocol.ECDHPrivateKey, cacheSize int) (*Proxy, error) {
-	return &Proxy{
-		Timeout:    DefaultTimeout,
-		commandKey: skey,
-		sessions:   cache.New(cacheSize),
-	}, nil
+// Vehicles must have the public part of skey enrolled on their keychains.
+// (This is a command-authentication key, not a TLS key.)
+func New(ctx context.Context, skey protocol.ECDHPrivateKey, cacheSize int, accountProvider account.AccountProvider) (*Proxy, error) {
```

Comment:
will it ever return error?

Reply:
Not currently. It may be plausible we will want it to in the future though... if adding validation for cache size or private key for example.

Reply:
yeah. I vote to remove it. We should be easily be able to refactor it later